web/web/files--common/modules/js/edit/PageEditModule.js

```diff
@@ -569,7 +569,7 @@ Wikijump.modules.PageEditModule.utils = {
 
 Wikijump.modules.PageEditModule.init = function(){
 	if(Wikijump.page.vars.locked == true){
-		Wikijump.utils.formatDates();
+		OZONE.utils.formatDates();
```

Comment:
Just verifying, we need this to reference an `OZONE` object?

Reply:
Yep, that's where the non-deprecated `formatDates` is: https://github.com/scpwiki/wikijump/blob/3f2efdaf889a03e797cc14e6cc875c08ab35d4f2/web/web/files--common/javascript/OZONE/utils.ts#L231